features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```
@@ -314,6 +314,9 @@
   "authentication.jit_provisioning.username_provisioning_url": "/accountrecoveryendpoint/register.do",
   "authentication.jit_provisioning.password_provisioning_url": "/accountrecoveryendpoint/signup.do",
 
+  "LegacyFeatures.EnableOAuth1": false,
```

```suggestion
  "legacyFeatures.enableOAuth1": false,
```
As lowercase has been used throughout the default.json